components/card/card.css

```diff
@@ -0,0 +1,94 @@
+@import "../../css/variables.css";
+
+main {
+    margin: 1rem 1rem;
```

Comment:
instead of using main tag selector, you can give class to main, because when the user will import your CL, his main tag will get this styling, and it'll mess up. Happened this with me last week.